spec/middleman-prismic/commands/prismic_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "spec_helper"
+
+describe Middleman::Cli::Prismic do
+  describe "#prismic" do
+    it "creates the directory that prismic data should live in" do
+      stub_options
+      stub_prismic_api
+
+      within_temp_directory do
```

Comment:
Does the current directory affect the calls outside of this block?

If not, what do you think about extracting out an `around` block?

```rb
around do |example|
  within_temp_directory { example.run }
end
```

If you're opposed to `around` blocks, what do you think about wrapping the entire test in the block so that tests can be more [clearly separated into phases with newlines](https://robots.thoughtbot.com/four-phase-test)?